src/Rpc/RpcClient.php

```diff
@@ -348,6 +351,25 @@ public function getDictionaryItemByURef(
         return StoredValueSerializer::fromJson($response['stored_value']);
     }
 
+    /**
+     * @throws RpcError
+     */
+    public function getGlobalState(string $blockHash, string $key, array $path = []): GlobalState
```

Comment:
Let's rename to `getGlobalStateByBlock` and add another one `getGlobalStateByStateRootHash`

Reply:
In Python SDK they added an abstraction called `GlobalStateID`:
- https://github.com/casper-network/casper-python-sdk/blob/bb11fa2fa708d17016562152596c71aa4fca9adc/pycspr/types/identifiers.py#L24
- https://github.com/casper-network/casper-python-sdk/blob/bb11fa2fa708d17016562152596c71aa4fca9adc/pycspr/api/client.py#L462

However, with only two possible values, having two separate methods will provide a simpler API for consumers